src/util/getRegion.js

```diff
@@ -6,8 +6,10 @@ const asyncReadFile = promisify(readFile);
 const homedir = os.homedir();
 
 const getRegionFromConfigStr = (configStr) => {
-  const defaultProfile = configStr.split('[').find(el => el.match('default'));
-  const regionLine = defaultProfile.split('\n').find(el => el.match('region'));
+  const defaultProfile = configStr.split('[').find((el) => el.match('default'));
```

Comment:
Interesting that we would have to put `split` and `find` on new lines when defining `regionLine` below but not for `defaultProfile`.